README.md

```
@@ -1,2 +1,67 @@
-# protocol-plugins
-Optional smart contract plugins: strategists, deposit access controllers, and external depositor helpers.
+<div align="center">
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)